library/std/src/f32.rs

```diff
@@ -87,6 +87,26 @@ impl f32 {
         unsafe { intrinsics::roundf32(self) }
     }
 
+    /// Returns the nearest integer to a number. Round half-way cases to the
+    /// nearest even integer.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let f = 3.3_f32;
+    /// let g = -3.3_f32;
+    ///
+    /// assert_eq!(f.round(), 3.0);
+    /// assert_eq!(g.round(), -3.0);
```

Comment:
This example uses `round`, not `round_to_even`. Would be good to show some half-way cases here.